.gitignore

```diff
@@ -0,0 +1 @@
+text.md
```

Comment:
Here you had to put the text.md file inside the .gitignore file, and the "This file should be ignored by github" was to be in the text.md

Reply:
No, the gitignore is a file where you list down the unnecessary files that should be ignored by git